crypto/constants.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+
+
+class TransactionTypes(Enum):
+    transfer = 0
+    second_signature_registration = 1
+    delegate_registration = 2
+    vote = 3
+    multi_signature_registration = 4
+    ipfs = 5
+    timelock_transfer = 6
+    multi_payment = 7
+    delegate_resignation = 8
+
+
+class TransactionFees(Enum):
```

Comment:
It's not used anywhere atm, but it will be once I start creating transaction builders, I will, however, change enums with a dict as per our discussion earlier.